src/middlewared/middlewared/plugins/vm/usb.py

```diff
@@ -10,7 +10,7 @@
 from .utils import get_virsh_command_args
 
 
-RE_VALID_USB_DEVICE = re.compile(r'^usb_\d+_\d+$')
+RE_VALID_USB_DEVICE = re.compile(r'^usb_\d+_\d+(_\d)*$')
```

Comment:
Forgot a + sign? (_\d+)

Reply:
```
bool(re.findall(r'^usb_\d+_\d+(_\d)*$', 'usb_5_1'))
True
bool(re.findall(r'^usb_\d+_\d+(_\d)*$', 'usb_5_1_'))
False
bool(re.findall(r'^usb_\d+_\d+(_\d)*$', 'usb_5_1_1'))
True
```

It is good as the whole group has `*` so `_\d` will be ensured but i think i should make it a non-capturing group. Will make that change :P

Reply:
```
>>> bool(re.findall(r'^usb_\d+_\d+(_\d)*$', 'usb_5_1_15'))
False
```

Reply:
Ah shoot :D Like that 😂